core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -205,8 +205,6 @@ private boolean anyNull(List<RexNode> operands) {
     map.put(SqlKind.IS_NOT_TRUE, Policy.NOT_NULL);
     map.put(SqlKind.IS_FALSE, Policy.NOT_NULL);
     map.put(SqlKind.IS_NOT_FALSE, Policy.NOT_NULL);
-    map.put(SqlKind.IS_DISTINCT_FROM, Policy.NOT_NULL);
```

Comment:
these changes differ from others in this review, can I get some comments regarding why these could be removed ?

Reply:
Here's the commit: https://github.com/apache/calcite/pull/834/commits/88e1d3301d20b37d1165f02d57b9f22d3437cf9e

It is named `Remove duplicate keys from Strong#MAP`

There were duplicate map.put lines, so I removed the extra ones.